src/common/devtools.js

```diff
@@ -145,6 +150,25 @@ const report =
     )
   ];
 
+const snapshot =
+  (model) => {
+    const id = Date.now().toString(32)
+    const message =
+      [ `\n\n`
+        , `echo '${JSON.stringify(window.application.model.value)}' > `
+        , `./dist/snapshot-${id}.json`
+        , ` && `
+        , `servo -w -b --pref dom.mozbrowser.enabled http://localhost:6060\?replay\=snapshot-${id}.json`
```

Comment:
> Up to you, but I think it's not useful to dump "echo" and "servo". What about just dump the JSON?

No preference, only reason I chose this was because first time I stored generated json file into root instead of `./dist` and it took me a while to figure why it did not worked, I though that would eliminate that confusion. I can switch back to just dumping `json` if you like.

In fact in longer term I wanted to just add a hook to the server that already serves files that would allow my to save dumps as json on a disk.